ui/src/present.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright © 2021 The Radicle Upstream Contributors
+//
+// This file is part of radicle-upstream, distributed under the GPLv3
+// with Radicle Linking Exception. For full terms see the included
+// LICENSE file.
+
+function shorten(
+  value: string,
+  length: number = 5,
+  offset: number = 2
+): string {
+  return `${value.slice(0, length + offset)}…${value.slice(-length)}`;
+}
+
+export const string = shorten;
```

Comment:
Exporting this as `string` instead of shorten is a bit confusing. Just looking at the method name I would not expect this to shorten the value.

Reply:
ede1ef8fc2128e8df4f4e717bfc45bdead41ed68